host/4/bullseye/amd64/python/python310/python310-composite.template

```diff
@@ -54,7 +54,9 @@ RUN apt-get update && \
     #  OpenMP dependencies
     apt-get install -y libgomp1 && \
     # mysql dependencies
-    apt-get install -y default-libmysqlclient-dev
+    apt-get install -y default-libmysqlclient-dev && \
+    #  Azure ML dependencies
+    apt-get install -y liblttng-ust0
```

Comment:
You need to include the && \ to continue the run statement on the next line. I believe. I will check out the builds

Reply:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/12449837/215843231-0622ccd2-5ace-446b-bb02-0164a6671429.png">
last lines are commented out